contracts/ModaPoolBase.sol

```diff
@@ -336,7 +335,7 @@ abstract contract ModaPoolBase is
 	function setWeight(uint32 _weight) external override {
 		// Only the factory can send a set weight command
 		require(msg.sender == address(modaPoolFactory), 'Access denied: factory only');
-		
+
```

Comment:
My IDE automatically removes bad white space on save. No changes other than that...